code/modules/mob/living/carbon/carbon.dm

```diff
@@ -212,11 +212,20 @@
 
 	var/atom/movable/thrown_thing
 	var/obj/item/I = get_active_held_item()
+	var/throw_range_modifier = 1 //Any modifiers we might apply to the throw range
 
 	if(!I || (I.flags_item & NODROP)) return
 
 	var/spin_throw = TRUE
 
+	if(mob_size == MOB_SIZE_BIG) //If we're big, we can throw further
+		throw_range_modifier *= 1.5
```

Comment:
Instead of mob size big check if its smaller or bigger than the mob throwing.

Reply:
No. If it's larger, it's harder regardless of relative size; this was the case for grapple toss and it will be the case here.

Reply:
This is pretty much tokenism anyways as I don't know of any big mobs that can actually default throw things; Warrior isn't big.

Reply:
Its quite outside of gameplay but, xenos can reinforve grab i believe so a big xeno can throw, and there are smaller mobs than humans(i think monkeys and maybe something someone may add), althought all xenos are bigger than humans by default.